src-frontend/app/adapters/application.js

```diff
@@ -0,0 +1,5 @@
+import DS from 'ember-data';
+
+export default DS.RESTAdapter.extend({
+  host: 'https://gentle-cove-4339.herokuapp.com'
```

Comment:
can we define all of these constants in an external file and then just do `import constants`?